src/components/Tabs/Tabs.js

```diff
@@ -60,7 +60,7 @@ export default class Tabs extends React.Component {
       });
 
 
-    const tabList = <ul className={`chi-tabs ${this.props.style === 'solid' ? '-solid -border' : '-border'} ${this.props.size ? `-${this.props.size}` : ''}`} id={this.state.id} role="tablist">
+    const tabList = <ul className={`chi-tabs ${this.props.vertical ? '-vertical' : '-border'} ${this.props.style === 'solid' ? '-solid' : ''} ${this.props.size ? `-${this.props.size}` : ''}`} id={this.state.id} role="tablist">
```

Comment:
```suggestion
    const tabList = <ul className=
    {`
      ${TABS_CLASSES.TABS} 
      ${this.props.vertical ? '-vertical' : '-border'} 
      ${this.props.style === 'solid' ? '-solid' : ''} 
      ${this.props.size ? `-${this.props.size}` : ''}
    `} 
    id={this.state.id} 
    role="tablist">
```
It would be great if we gradually add variables for classes to all components, here for example: `TABS_CLASSES`.